src/compiler/core.ts

```diff
@@ -1237,20 +1237,20 @@ namespace ts {
         getSignatureConstructor(): new (checker: TypeChecker) => Signature;
     }
 
-    function Symbol(flags: SymbolFlags, name: string) {
+    function Symbol(this: Symbol, flags: SymbolFlags, name: string) {

```

Comment:
@rakatyal just remembered your tslint rule would have issues with this because it can't distinguish between `ts.Symbol` and `Symbol`